### ISSUE TITLE
Getting a 500 Internal Server Error when trying to authenticate

### ISSUE THREAD

**Issue by OmElDonia** [heviat/Mailu#2]
### Environment

- [x] docker compose

## Description

I have set up OpenID Connect on Authentik, when getting redirected after successfully putting in my credentials it throws a 500 error on redirection.

## Logs

```
ERROR in app: Exception on /sso/login [GET]
Traceback (most recent call last):
  File "/app/venv/lib/python3.10/site-packages/flask/app.py", line 2525, in wsgi_app
    response = self.full_dispatch_request()
  File "/app/venv/lib/python3.10/site-packages/flask/app.py", line 1822, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/app/venv/lib/python3.10/site-packages/flask/app.py", line 1820, in full_dispatch_request
    rv = self.dispatch_request()
  File "/app/venv/lib/python3.10/site-packages/flask/app.py", line 1796, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/app/mailu/sso/views/base.py", line 26, in login
    username, sub, id_token, token_response = utils.oic_client.exchange_code(flask.request.query_string.decode())
  File "/app/mailu/utils.py", line 195, in exchange_code
    return user_response['email'], user_response['sub'], response["id_token_jwt"], response
  File "/app/venv/lib/python3.10/site-packages/oic/oauth2/message.py", line 762, in __getitem__
    return self._dict[item]
KeyError: 'email'
```

**Comment by OmElDonia** [comment]
After some tinkering around, I found the issue. It seems like in `get_redirect_url`, the only scope requested is `openid`, while we also need `email`, as OpenID Connect requires it [per specification](https://openid.net/specs/openid-connect-core-1_0.html#ScopeClaims) to get the `email` claim.

I have submitted a very simple PR to fix this.

**Comment by Encotric** [comment]
Hi, sorry for the (very) late response, and thank you for your work! I merged your PR into `staging` (6ad3dcb) and will create a new `master` release after some testing of the merged updates made by the upstream Mailu repository.
